ci/snaps/snaps_os_tmplt.yaml

```diff
@@ -0,0 +1,550 @@
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at:
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+---
+openstack:
+  connections:
+    # Note - when http_proxy is set, you must also configure ssh for proxy tunneling on your host.
+    - connection:
+        name: admin-creds
+        username: {{ admin_user }}
+        project_name: {{ admin_proj }}
+        password: {{ admin_pass }}
+        auth_url: {{ auth_url }}
+        identity_api_version: {{ id_api_version }}
+        proxy_settings:
+          host: {{ proxy_host }}
+          port: {{ proxy_port }}
+          ssh_proxy_cmd: {{ ssh_proxy_cmd }}
+  projects:
+    - project:
+        os_creds_name: admin-creds
+        name: os-deploy-proj-{{ build_id }}
+        description: Project for Orchestrators
+        users: [os-deploy-user-{{ build_id }}, {{ admin_user }}]
+  users:
+    - user:
+        os_creds_name: admin-creds
+        name: os-deploy-user-{{ build_id }}
+        password: {{ os_user_pass }}
+        project_name: os-deploy-proj-{{ build_id }}
+        roles: {admin: os-deploy-proj-{{ build_id }}}
+  flavors:
+    - flavor:
+        os_creds_name: admin-creds
+        name: os-build-flavor-{{ build_id }}
+        ram: 4096
+        disk: 80
+        vcpus: 2
+    - flavor:
+        os_creds_name: admin-creds
+        name: os-node-flavor-{{ build_id }}
+        ram: 8192
+        disk: 100
+        vcpus: 4
+  images:
+    - image:
+        os_creds_name: admin-creds
+        name: snaps-os-image-{{ build_id }}
+        format: qcow2
+        public: True
+        image_user: ubuntu
+        download_url: https://cloud-images.ubuntu.com/xenial/current/xenial-server-cloudimg-amd64-disk1.img
+  networks:
+    - network:
+        os_user:
+          name: os-deploy-user-{{ build_id }}
+          project_name: os-deploy-proj-{{ build_id }}
+        name: os-ctrl-net-{{ build_id }}
+        project_name: os-deploy-proj-{{ build_id }}
+        subnets:
+          - subnet:
+              name: os-ctrl-subnet-{{ build_id }}
+              project_name: os-deploy-proj-{{ build_id }}
+              cidr: {{ ctrl_ip_prfx }}.0/24
+              dns_nameservers: [8.8.8.8]
+    - network:
+        os_user:
+          name: os-deploy-user-{{ build_id }}
+          project_name: os-deploy-proj-{{ build_id }}
+        name: os-admin-net-{{ build_id }}
+        project_name: os-deploy-proj-{{ build_id }}
+        subnets:
+          - subnet:
+              name: os-admin-subnet-{{ build_id }}
```

Comment:
Disable the gateway for this subnet.

Reply:
How do you do that? The gateway IP has not been configured.

Reply:
Neutron API does not appear to allow for what is being requested. Neutron API does not appear to allow for what is being requested.

Reply:
Here’s the link to the Network API (https://developer.openstack.org/api-ref/network/v2/index.html). Under Post for Subnets, the only attribute I see that would control this value is gateway_ip which is null. See the description below:

"Gateway IP of this subnet. If the value is null that implies no gateway is associated with the subnet. If the gateway_ip is not specified, OpenStack Networking allocates an address from the CIDR for the gateway for the subnet by default."